hadoop-ozone/dist/src/main/smoketest/omha/om-roles.robot

```diff
@@ -28,6 +28,9 @@ Assert Leader Present in JSON
      [Arguments]                     ${output}
      ${leader} =                     Execute                        echo '${output}' | jq '.[] | select(.[] | .serverRole == "LEADER")'
                                      Should Not Be Equal            ${leader}       ${EMPTY}
+Assert Leader Present in TABLE
+     [Arguments]                     ${output}
+     Should Match Regexp             ${output}                     \|\s+om\d+\s+\|\s+LEADER\s+\|
```

Comment:
1. `om host`  is a user-defined string and does not necessarily start with `om`, right ?
2. NIT: string don't look aligned.
3. I verified this RE, and it seems that any format will pass.  So, this RE looks like it needs to be changed.
You can install robot in python and execute it locally:
`om-roles-test.robot` :
```shell

*** Settings ***
Documentation       Smoke test for listing om roles.
Library             OperatingSystem
Test Timeout        5 minutes

*** Keywords ***
Assert Leader Present in TABLE
     [Arguments]                     ${output}
     Should Match Regexp             ${output}      \|\s+om\d+\s+\|\s+LEADER\s+\|

*** Test Cases ***

List om roles as TABLE without OM service ID passed
    ${result}=        Set Variable       | host32 | om32 | OTHER
    Should Be Equal   ${result}          | host32 | om32 | OTHER
    Assert Leader Present in TABLE  ${result}
```
run: `robot om-roles-test.robot`
```
==============================================================================
Om-Roles-Test :: Smoke test for listing om roles.
==============================================================================
List om roles as TABLE without OM service ID passed                   | PASS |
------------------------------------------------------------------------------
Om-Roles-Test :: Smoke test for listing om roles.                     | PASS |
1 test, 1 passed, 0 failed
==============================================================================
```

Reply:
> om host is a user-defined string and does not necessarily start with om, right ?

This regex matches strings containing "om" followed by digits, specifically formatted with "LEADER" surrounded by pipe delimiters. 

![image](https://github.com/user-attachments/assets/f9197b0f-7ee7-408c-af05-fc11411e3a8a)